lms/djangoapps/branding/api.py

```diff
@@ -261,7 +261,11 @@ def _footer_logo_img(is_secure):
         Absolute url to logo
     """
     logo_name = microsite.get_value('FOOTER_ORGANIZATION_IMAGE', settings.FOOTER_ORGANIZATION_IMAGE)
-    return _absolute_url_staticfile(is_secure, logo_name)
+    try:
+        return _absolute_url_staticfile(is_secure, logo_name)
+    except ValueError:
```

Comment:
I'm not sure that I understand this. If the specified footer image doesn't exist, isn't that a misconfiguration? For the particular issue we are seeing on sandboxes, why is the configuration to a file that doesn't exist? Wouldn't the fix be to change the configuration to point to a file that does exist?

It also seems unfortunate if every place where we look up a staticfile we have to have a fall-through handler. What is special about this case? Should you add such logic to every other place that we do this? Maybe there should be a microsite image accessor that does the fall-through logic in one place so that we don't have to repeat this.

I can see that maybe it is better for the developer for the site not to go down in this situation, but to use a fallback image. However, in that case I would expect to log this so that it is possible to diagnose why the configured image is not being shown. 

Also, I thought that theming already had fall-through logic such that it would access the global version if the theme version wasn't found.